src/dms-preview/azext_dms/custom.py

```diff
@@ -69,90 +82,85 @@ def create_or_update_project(
                                    group_name=resource_group_name,
                                    service_name=service_name,
                                    project_name=project_name)
+
+
+def get_project_platforms(cmd, project_name, service_name, resource_group_name):
+    client = dms_cf_projects(cmd.cli_ctx)
+    proj = client.get(group_name=resource_group_name, service_name=service_name, project_name=project_name)
+    return (proj.source_platform.lower(), proj.target_platform.lower())
 # endregion
 
 
 # region Task
 def create_task(
+        cmd,
         client,
         resource_group_name,
         service_name,
         project_name,
         task_name,
-        source_platform,
-        target_platform,
         task_type,
         source_connection_json,
         target_connection_json,
         database_options_json,
         enable_schema_validation=False,
         enable_data_integrity_validation=False,
-        enable_query_analysis_validation=False):
-
-    # Set inputs to lowercase
+        enable_query_analysis_validation=False,
+        validate_only=False):
+
+    # Get source and target platform abd set inputs to lowercase
+    src, tgt = get_project_platforms(cmd,
```

Comment:
To be consistent with other naming conventions present, can you rename src and tgt to source_platform and target_platform, respectively?

Reply:
Done.